BDKSwiftExampleWallet/Utilities/Constants.swift

```diff
@@ -16,8 +16,8 @@ struct Constants {
                 private static let blockstream = "https://blockstream.info/api"
                 private static let mempoolspace = "https://mempool.space/api"
                 static let allValues = [
-                    mempoolspace,
                     blockstream,
+                    mempoolspace,
```

Comment:
this doesnt seem like its needed anymore since it just moves `mempoolspace` from one place to another